.github/dependabot.yml

```diff
@@ -0,0 +1,195 @@
+version: 2
+registries:
+  maven-repository-nexus-intern-sda-se-online-repository-sda-se-re:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online/repository/sda-se-releases
+    username: SDABot
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_REPOSITORY_SDA_SE_RE_PASSWORD}}"
+  maven-repository-nexus-intern-sda-se-online-443-repository-sda-s:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online:443/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_443_REPOSITORY_SDA_S_PASSWORD}}"
+  maven-repository-nexus-sda-se-io-443-repository-sda-se-public:
+    type: maven-repository
+    url: https://nexus.sda-se.io:443/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_SDA_SE_IO_443_REPOSITORY_SDA_SE_PUBLIC_PASSWORD}}"
+  maven-repository-nexus-intern-sda-se-online-repository-sda-se-pu:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_REPOSITORY_SDA_SE_PU_PASSWORD}}"
+  maven-repository-nexus-sda-se-io-repository-sda-se-public:
+    type: maven-repository
+    url: https://nexus.sda-se.io/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_SDA_SE_IO_REPOSITORY_SDA_SE_PUBLIC_PASSWORD}}"
+
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "02:00"
+    timezone: Europe/Berlin
+  open-pull-requests-limit: 33
+  target-branch: master
+  ignore:
+  - dependency-name: org.scala-lang:scala-library
+    versions:
+    - ">= 2.13.a, < 2.14"
+  - dependency-name: org.scala-lang:scala-reflect
+    versions:
+    - ">= 2.13.a, < 2.14"
```

Comment:
I think these are the only relevant exclusions as we don't want to leave the scala 2.12 channel. The remaining (aws, flyway, ...) should be removed.